api/cloud/oc_cloud_apis.c

```diff
@@ -276,6 +276,7 @@ oc_cloud_deregister(oc_cloud_context_t *ctx, oc_cloud_cb_t cb, void *data)
 
     OC_DBG("try deregister");
     bool cannotConnect = true;
+	cloud_set_cps(ctx, OC_CPS_DEREGISTERING);
```

Comment:
[FYI] there seems to be a bit of inconsistent formatting in these changes, no reason to block merge because of that, but its something I've noticed. 

There is a clang-format files in the [tools folder](https://github.com/iotivity/iotivity-lite/tree/master/tools), so formatting can be automated and enforced. @WAvdBeek whatabout running clang-format over the whole repo and adding a workflow to check formatting in GitHub?

Reply:
it was disabled since it was taking more time to fix formatting than the actual code.

Reply:
I am confused, does the formatting need corrections or something? Because it can be fully automated by installing the pre-commit hook locally, clang will handle the formatting automatically then and nothing needs to be done by the programmer.

Reply:
no worries about the formatting, currently not checked.

next week I will try to get (someone) the automatic formatter to work on the code base.